.github/workflows/regression.yml

```diff
@@ -128,7 +128,7 @@ jobs:
         uses: docker://ghcr.io/su2code/su2/test-su2:220614-1237
         with:
           # -t <Tutorials-branch> -c <Testcases-branch>
-          args: -b ${{github.ref}} -t develop -c develop -s ${{matrix.testscript}}
+          args: -b ${{github.ref}} -t develop -c feature_FFD -s ${{matrix.testscript}}
```

Comment:
I know we talked about this before, but I am making a note here  just to be sure.

I forgot to change this back in a recent PR.